tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmCapabilitiesIndices.cs

```diff
@@ -45,6 +45,10 @@ internal static class RcmCapabilitiesIndices
 
         public static readonly BigInteger AsmCustomDataScanners = Create(17);
 
+        public static readonly BigInteger AsmExclusionData = Create(18);
+
+        public static readonly BigInteger ApmTracingTracingEnabled = Create(19);
```

Comment:
`TracingTracing` looks weird, but we're following a `{ProductName}{SettingName}` convention.

Reply:
It's nothing compared to `AutoInstrumentation.ManualInstrumentation` 😅